ods_ci/tests/Tests/600__ai_apps/680__starburst/682__starburst_enterprise_sm.robot

```diff
@@ -0,0 +1,158 @@
+${QUERY_CATALOGS}=    SHOW CATALOGS
+${QUERY_SCHEMAS}=    SHOW SCHEMAS from tpch
+${QUERY_TABLES}=    SHOW TABLES from tpch.sf1
+${QUERY_CUSTOMERS}=    SELECT name FROM tpch.sf1.customer limit 3
+${QUERY_JOIN}=    SELECT c.name FROM tpch.sf1.customer c JOIN tpch.sf1.orders o ON c.custkey = o.custkey ORDER BY o.orderdate DESC limit 3    # robocop: disable
+${QUERY_CATALOGS_PY}=    sql = '${QUERY_CATALOGS}'\ndf = get_sql(sql, conn)\nprint(df['Catalog'].values)\n    # robocop: disable
+${QUERY_SCHEMAS_PY}=    sql = '${QUERY_SCHEMAS}'\ndf = get_sql(sql, conn)\nprint(df['Schema'].values)\n    # robocop: disable
+${QUERY_TABLES_PY}=    sql = '${QUERY_TABLES}'\ndf = get_sql(sql, conn)\nprint(df['Table'].values)\n
+${QUERY_CUSTOMERS_PY}=    sql = '${QUERY_CUSTOMERS}'\ndf = get_sql(sql, conn)\nprint(df['name'].values)\n    # robocop: disable
+${QUERY_JOIN_PY}=    sql = '${QUERY_JOIN}'\ndf = get_sql(sql, conn)\nprint(df['name'].values)\n    # robocop: disable
+
+
+*** Test Cases ***
+Verify Starburst Enterprise Operator Can Be Installed
+    [Documentation]    Installs Starburst enterprise operator and check if
+    ...                its tile/card appears in RHODS Enabled page
+    [Tags]    ODS-2247    Tier2
+    Log    message=Operator is installed and CR is deployed as part of Suite Setup
+    Verify Service Is Enabled         ${SEP_DISPLAYED_NAME}
+
+Verify User Can Perform Basic Queries Against Starburst From A DS Workbench    # robocop: disable
+    [Documentation]    Creates, launches a DS Workbench and query the Starburst
+    ...                default catalogs
+    [Tags]    ODS-2249    Tier2
+    [Setup]    Create Route And Workbench
+    Launch And Access Workbench    workbench_title=${DS_WORKBENCH_NAME}
+    ...    username=${TEST_USER_3.USERNAME}     password=${TEST_USER_3.PASSWORD}
+    ...    auth_type=${TEST_USER_3.AUTH_TYPE}
+    Open New Notebook In Jupyterlab Menu
+    Run Cell And Check For Errors    !pip install pandas;pip install trino
+    Run Cell And Check For Errors    ${GET_SQL_FUNC}
+    Run Cell And Check For Errors    ${INIT_CELL_CODE}
+    Run Query And Check Output    query_code=${QUERY_CATALOGS_PY}
+    ...    expected_output=['system' 'tpch']
+    Run Query And Check Output    query_code=${QUERY_SCHEMAS_PY}
+    ...    expected_output=['information_schema' 'sf1' 'sf100' 'sf1000' 'sf10000' 'sf100000' 'sf300' 'sf3000' 'sf30000' 'tiny']    # robocop: disable
+    Run Query And Check Output    query_code=${QUERY_TABLES_PY}
+    ...    expected_output=['customer' 'lineitem' 'nation' 'orders' 'part' 'partsupp' 'region' 'supplier']    # robocop: disable
+    Run Query And Check Output    query_code=${QUERY_CUSTOMERS_PY}
+    ...    expected_output=('Customer#[0-9]+'\s?)+
+    ...    use_regex=${TRUE}
+    Run Query And Check Output    query_code=${QUERY_JOIN_PY}
+    ...    expected_output=('Customer#[0-9]+'\s?)+
+    ...    use_regex=${TRUE}
+    Capture Page Screenshot
```

Comment:
I've been thinking that for code cells that might be hidden when taking a screenshot of the page (because they are further down in the webpage/notebook, especially after output has been printed) we should try using `Capture Element Screenshot` instead (https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Capture%20Element%20Screenshot). Not needed to add it here but thought I'd share.

Reply:
valid point. I think we should try using that kw. 
However, in this particular case the page screeenshot captures all the queries
![image](https://github.com/red-hat-data-services/ods-ci/assets/88311595/657865ab-78da-427f-ae51-ab2b12de504a)